tests/tests.cmake

```
@@ -3,6 +3,8 @@
 #
 target_sources(testprecice
     PRIVATE
+    tests/geomultiscale-mapping/AxialGeoMultiscale.cpp
```

```suggestion
    tests/geometric-multiscale/AxialGeoMultiscale.cpp
```
would style-wise be more fitting and neither too long